### PR TITLE
Added Group `id`

### PR DESCRIPTION
**Pull request by RichardLitt**
I was unable to get `&format` working. Any ideas?

### CODE REVIEW

**Comment by RichardLitt** [comment]
cc @whyrusleeping

File: apiary.apib [hunk 1]
```diff
@@ -990,6 +990,199 @@ Outputs the list of peers that were removed.
...
+if no peer is specified, prints out local peers info.
+
+'ipfs id' supports the format option for output with the following keys:
+<id> : the peers id
+<aver>: agent version
+<pver>: protocol version
+<pubkey>: public key
+<addrs>: addresses (newline delimited)
+
+## id [GET /id{?arg}{&format}]
```
Comment (daviddias) [inline apiary.apib]: Wouldn't it be better if the format is something that the cli understand and transforms the json coming from the API?

**Comment by daviddias** [comment]
Other than the `format` note, LGTM :)

File: apiary.apib (hunk 1 above)
Comment (RichardLitt) [inline apiary.apib]: TBH, I can't get the `format` tag working at all! 

Example that should work:

```
curl -i -X POST http://localhost:5001/api/v0/id?arg=QmepgFW7BHEtU4pZJdxaNiv75mKLLRQnPi1KaaXmQN4V1a&format="<addrs>"
```

Comment (daviddias) [inline apiary.apib]: @whyrusleeping is the format really part of the API? It should be just the cli that prints out depending on the requested format, right?

Comment (RichardLitt) [inline apiary.apib]: ping @whyrusleeping

**Comment by RichardLitt** [comment]
Needs to be checked before #20 can be closed.

**Comment by whyrusleeping** [comment]
It looks like format is only used client side to format the json blob we get back.

**Pull request merged**